resources/migrations/000_migrations.yaml

```diff
@@ -13073,6 +13073,61 @@ databaseChangeLog:
                     nullable: false
                   defaultValueComputed: current_timestamp
 
+  - changeSet:
+      id: v46.00-028
+      author: tsmacdonald
+      comment: Added 0.46.0 -- Join table connecting cards to dashboards/cards's parameters that need custom filter values from the card
+      changes:
+        - createTable:
+            tableName: values_card
+            remarks: "Join table connecting cards to entities (dashboards, other cards, etc.) that use the values generated by the card for filter values"
+            columns:
+              - column:
+                  name: id
+                  type: int
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: updated_at
+                  type: ${timestamp_type}
+                  defaultValueComputed: current_timestamp
+                  remarks: "most recent modification time"
+                  constraints:
+                    nullable: false
+              - column:
+                  name: created_at
+                  type: ${timestamp_type}
+                  defaultValueComputed: current_timestamp
+                  remarks: "creation time"
+                  constraints:
+                    nullable: false
+              - column:
+                  name: card_id
```

Comment:
I'm surprised to see this is called `card_id` but isn't a FK to `report_card`.

Reply:
(see comment thread here: https://github.com/metabase/metabase/pull/27008#discussion_r1050821108 )